frontend/packages/design-system/src/components/Accordion/Accordion.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, ReactNode } from 'react';
+import { Text, Box, Pressable } from 'native-base';
+
+type Props = {
+  header: string;
+  content: ReactNode;
+};
+
+export const AccordionComponent: FC<Props> = ({ header, content }) => {
+  const [isActive, setIsActive] = React.useState(false);
+
+  return (
+    <Box>
+      <Pressable bg="neutral.200" p="2" onPress={() => setIsActive(!isActive)}>
+        <Text variant="heading">{header}</Text>
+      </Pressable>
+      {isActive && (
+        <Box bg="neutral.100" p="2">
```

Comment:
Agree about the color, but if the header has a padding, I'd keep the padding here too, to ensure consistency. 
I wouldn't leave it up to the user of this component to know what the padding of the header is exactly and set the same on their content.
I think more padding is acceptable, but not less, so I'd keep this as a minimum.

Reply:
Yeah that sounds good

Reply:
okay, so setting the padding in the DS and bg color in the actual app is problematic, color and padding need to be on the same element.
I thought about how to solve this, and I arrived at the conclusion, that, since this is the design system, there will be some common styling to the children, like background color of the box.